products/surface.md

```diff
@@ -10,71 +10,117 @@ latestColumn: true
 eolColumn: End of Servicing Date
 releaseColumn: false
 releaseDateColumn: true
-sortReleasesBy: 'release'
+sortReleasesBy: eol
```

Comment:
@captn3m0 i know ( at least guessing)  this is working but isnt this tag needs to be a string ? i mean isnt it needs " " ?